### ISSUE TITLE
Include babel-core as peer dependency (or bundle with output)

### ISSUE THREAD

**Issue by jkjustjoshing** [tommoor/promise-pool#4]
Right now I get the following error when trying to use the library:

<img width="505" alt="image" src="https://user-images.githubusercontent.com/813192/71688093-36a9bc00-2d6d-11ea-83f7-8275e254ac5f.png">

I think this is because `babel-core` is a devDependency, not a dependency/peerDependency.

**Comment by brandly** [comment]
i'm seeing this too on nodejs v12.13.0. the published package begins with a series of `require`s that'll currently fail.

```js
var _regenerator = require("babel-runtime/regenerator");

var _regenerator2 = _interopRequireDefault(_regenerator);

var _asyncToGenerator2 = require("babel-runtime/helpers/asyncToGenerator");

var _asyncToGenerator3 = _interopRequireDefault(_asyncToGenerator2);

var _classCallCheck2 = require("babel-runtime/helpers/classCallCheck");

var _classCallCheck3 = _interopRequireDefault(_classCallCheck2);

var _createClass2 = require("babel-runtime/helpers/createClass");

var _createClass3 = _interopRequireDefault(_createClass2);

var _promise = require("babel-runtime/core-js/promise");

var _promise2 = _interopRequireDefault(_promise);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```

i took the source, stripped the flow types by hand, and ran it without any additional runtime. it [might be worth considering](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#Browser_compatibility) just leaving the async functions in the published file!

p.s. thanks for this repo! it's exactly what i was looking for.

**Comment by tommoor** [comment]
Should be resolved by `v1.0.3`

**Comment by brandly** [comment]
thank you! the package is working for me now.
